src/components/PageBlocks/base.vue

```diff
@@ -57,10 +57,42 @@ export default {
     },
   },
 
+  data () {
+    return {
+      refreshInterval: null,
+      key: 0,
+    }
+  },
+
   computed: {
     options () {
       return this.block.options
     },
+
+    allowsRefresh () {
+      // TODO: Add record pages
+      const whitelisted = (route) => this.$route.name === route
+      return this.options.refreshRate >= 5 && ['page'].some(whitelisted)
```

Comment:
`['page'].includes(this.$route.name)` seems like what you wanted to use

Reply:
Yes. I need to speak to you a bit about this.